src/Components/SmartComponents/CVEDetailsPage/CVEDetailsPage.js

```diff
@@ -59,7 +59,10 @@ const CVEDetailsPage = (props) => {
     const cveStatusDetails = {
         id: cveName,
         status_id: cveDetails.data.status_id,
-        justification: cveDetails.data.status_justification
+        justification: cveDetails.data.status_justification,
+        exposed_systems_count: cveDetails.data.systems_status_detail
```

Comment:
There is another way to get that information. From Redux. We store all the information about exposed systems in `CveDetailsPageStore` 

You can get it from the component like this

```
  const exposedSystems = useSelector(
        ({ CVEDetailsPageStore }) => CVEDetailsPageStore.affectedSystemsByCVE
    );
```

and use 

```
        exposed_systems_count: exposedSystems?.payload?.meta?.total_items || 0

```

On the other hand, if you want to get it from cveDetails object. I believe it's a good practise to keep the component as "dump" as possible. You can "prepare" the data structure inside the [helper](https://github.com/RedHatInsights/vulnerability-ui/blob/master/src/Helpers/CVEHelper.js#L72)